content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,15 +240,20 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only that control plane, and the workloads in clusters managed by that control plane.
 
-- Configuration isolation: You can make configuration changes in one cluster,
-  zone, or region without impacting others.
+- Improved configuration isolation: Configuration changes in one control plane affects only
+the workloads in clusters managed by this control plane.
 
-You can improve control plane availability through fail over. When a control
-plane instance becomes unavailable, workload instances can connect to
-another available control plane instance. Fail over can happen across clusters,
-zones, or regions.
+#### Failover in Shared Control Planes
+
+You can improve shared control plane availability through fail over. When a shared control
```

Comment:
Does failover across cluster boundaries actually work today?

Reply:
I have absolutely no clue if it works. But it was listed in the doc (you can actually see it live on our site)!. So I am simply repeating that.. Heck it even has diagrams!

Reply:
I'd be in favor of removing it from the doc.

@smawson WDYT?